packages/protocol/test/layer1/based/InboxTest_ProposeAndProve.t.sol

```diff
@@ -522,18 +522,6 @@ contract InboxTest_ProposeAndProve is InboxTestBase {
         address operator = Bob;
         resolver.registerAddress(block.chainid, "inbox_operator", operator);
         vm.stopPrank();
-
-        vm.startPrank(Alice);
-        params.proposer = operator;
-        vm.expectRevert(ITaikoInbox.NotInboxWrapper.selector);
-        inbox.proposeBatch(abi.encode(params), "txList");
-        vm.stopPrank();
-
-        vm.startPrank(operator);
-        params.proposer = address(0);
-        vm.expectRevert(ITaikoInbox.CustomProposerMissing.selector);
-        inbox.proposeBatch(abi.encode(params), "txList");
-        vm.stopPrank();
```

Comment:
Since rn the wrapper address is an immutable value, so I have to delete these two tests for now.

Reply:
Sounds good. I will take a look at how to add them back.